innersource-license.md

```diff
@@ -0,0 +1,83 @@
+## Title
+
+InnerSource License
+
+## Patlet
+
+Two companies that belong to the same group want to share software source code with each other but they are concerned about the implications in terms of legal liabilities or cross-company accounting. An **InnerSource License** provides a reusable legal framework for the inter-company agreement within the group, which opens up new collaboration options.
+
+## Problem
+
+When two companies within a group want to share code with each other, they need an agreement about the terms and often a legal contract. Creating such agreements on a per project basis takes effort and creates a barrier for sharing. i.e. a team/company might decide not to share their source code with another company in the group because it seems complicated.
+
+Barriers for sharing can lead to silos and duplication of effort in rebuilding similar solutions in multiple parts of the group.
+
+At the time of sharing the source code, it can not be reliably predicted what the value of sharing will be. Hence if the activity of sharing has any significant cost, the companies are unlikely to do it, as they are concerned about the return on investment.
+
+## Context
+
+- A large group with many companies (subsidiaries) that want to share code. (when the group gets larger, the value of this pattern increases)
+- These companies are legally independent units.
```

Comment:
```suggestion
- These legal entities are entities with own legal rights, obligations
```

Reply:
I worked in something similar. Resolving.